sql/core/src/main/scala/org/apache/spark/sql/execution/objects.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.sql.execution
 
+import com.sun.org.apache.xalan.internal.xsltc.compiler.util.VoidType
```

Comment:
Yes, that is why the first try was failed. It was unintentionally imported during my debugging.